---
layout: page
title: Software
permalink: /code/
---

This page collects released software packages.  
The results in papers are generated by analysis scripts that are separate from released and carefully documented software packages.  It may not be as pretty, or unit tested, or documented, but nonetheless, in an effort towards best reproducible research, we are trying to release the code supporting published results.  Where available, these will be linked from the individual [papers](/papers).

## R packages 
To install R packages directly from github, do, in R:
```
install.packages("devtools")
library(devtools)
install_github("chr1swallace/package.name")
```
# coloc
[package website](/coloc) // 
[CRAN](http://cran.r-project.org/web/packages/coloc/index.html) // [github](https://github.com/chr1swallace/coloc) // 
A package of R functions used to test for
colocalisation of two genetic traits.  For more detail or
the statistical background and applications see:
{% cite 2013_genepi_coloc %}
{% cite 2014_plosgen_giambartolomei %}.

New in v3.1 which is now up to date on CRAN: `finemap.abf()` offers finemapping of single traits under a single causal variant assumption.  See [vignette](https://rawgit.com/chr1swallace/coloc/master/inst/doc/vignette.html) for details.

# GUESSFM
[package website](/GUESSFM) // 
[github](https://github.com/chr1swallace/GUESSFM) //
A wrapper around the GUESS stochastic model search method for fine
mapping causal variants in genetic data.   See {% cite 2015_il2ra %} [more detail here](/research).

# simGWAS
[package website](/simGWAS) // 
[github](https://github.com/chr1swallace/simGWAS) //
Fast and accurate simulation of case-control GWAS summary statistics {% cite Fortune313023 %}

# colocCommonControl
[CRAN](http://cran.r-project.org/web/packages/colocCommoncontrol/index.html) // [github](https://github.com/mdfortune/colocCommonControl) //
Adaptation of coloc for comparing two GWAS studies which share controls 
{% cite 2015_ng_fortune %}.

# annotSnpStats
[github](https://github.com/chr1swallace/annotSnpStats) //
extends SnpMatrix objects from the snpStats package by tying in
sample and snp annotation data.frames.  It allows the joint
manipulation, and binding, of such objects, and includes a few
functions I use for finding duplicate samples, applying QC, and
aligning alleles between different datasets.  It seems to work,
and I intend to push it to CRAN in the future.

# snpStatsWriter
[github](https://github.com/mdfortune/colocCommonControl) //
allows writing snpStats objects to disk in formats suitable
for reading by snphap, phase, mach, IMPUTE, beagle, and
(almost) anything else that expects a rectangular format.

# wgsea
[CRAN](http://cran.r-project.org/web/packages/wgsea/index.html) //
implements a
Wilcoxon-based test for Gene Set Enrichment Analysis when
testing regions of the genome according to GWAS derived
evidence for SNP-trait association.  For more detail, see
the supplementary methods of {% cite 2010_nat_heinig %}.

# extraBinomial
[CRAN](http://cran.r-project.org/web/packages/extraBinomial) //
(author Xin Yang, listed here because Xin doesn't have her own website)
uses an extra binomial model to test for case-control
allele frequency differences in pooled sequence data {% cite 2012_bioinformatics_yang %}.

# origin
[github](https://github.com/chr1swallace/origin) //
implements an   extension to the Weinberg method for testing for parent of origin
effects to allow for multiple affected offspring in a nuclear
family.  I haven't used it in a while, and it requires some checking
just to get it to pass the CRAN tests.  Unless I need to revisit it
for another reason, I am unlikely to push it to CRAN, but it is a
finished package.
For more detail see:

Weinberg et al. Methods for detection of parent-of-origin effects in
    genetic studies of case-parents triads. /Am J Hum Genet/
    65:229-235. 1999.
    
Wallace et al. The imprinted DLK1-MEG3 gene region on chromosome
14q32.2 alters susceptibility to type 1 diabetes. /Nat
Genet/ 2010.
[ [Europe PMC](http://europepmc.org/abstract/MED/19966805) ]

## Non-R packages
# VSEAMS (Olly Burren)
Variant set enrichment analysis (of association summary statistics) using multivariate sampling. Please see the wiki available at [this wiki](http://github.com/ollyburren/vseams/wiki) for further details.  Method described in 
{% cite 2014_bioinformatics_burren %}.

<!-- ### IBDSCORE  -->

<!-- implements a score test for the Rice-Holmans IBD -->
<!-- 	regression model. -->
<!-- 	The model is described by -->

<!--  Holmans, P. /Human Heredity/ 2002. 53:92-102 -->

<!-- Rice et al. /Genet Epidemiol/ 1999. 17:S691-S695 -->

<!-- and this score test by -->

<!-- Wallace et al. /Am J Hum Genet/ 2006. 79:323-31 [ [Europe PMC](http://europepmc.org/abstract/MED/16826522) ] {{{padlock}}} -->

<!-- [ [download](http://www-gene.cimr.cam.ac.uk/staff/wallace/soft/ibdscore.tgz) ] -->

## See also...

 [published R packages on CRAN](http://crantastic.org/authors/2111) // [all code at github](http://github.com/chr1swallace) // [code blog](http://cwcode.wordpress.com) 
