<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>Data</title>
<!-- 2017-01-25 Wed 21:12 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center; }
  .todo   { font-family: monospace; color: red; }
  .done   { color: green; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  pre.src-sh:before    { content: 'sh'; }
  pre.src-bash:before  { content: 'sh'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-R:before     { content: 'R'; }
  pre.src-perl:before  { content: 'Perl'; }
  pre.src-java:before  { content: 'Java'; }
  pre.src-sql:before   { content: 'SQL'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.right  { text-align: center;  }
  th.left   { text-align: center;   }
  th.center { text-align: center; }
  td.right  { text-align: right;  }
  td.left   { text-align: left;   }
  td.center { text-align: center; }
  dt { font-weight: bold; }
  .footpara:nth-child(2) { display: inline; }
  .footpara { display: block; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  /*]]>*/-->
</style>
<link rel="stylesheet" type="text/css" href="css/poole.css"/>
<link rel="stylesheet" type="text/css" href="css/lanyon.css"/>
<link rel="stylesheet" type="text/css" href="css/tabs.css"/>
<link rel="stylesheet" type="text/css" href="css/colours.css"/>
<script type="text/javascript" src="styles/bigblow/js/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery-ui-1.10.2.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.localscroll-min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.scrollTo-1.4.3.1-min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.zclip.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/bigblow.js"></script>
<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">Data</h1>
    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior.  -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">
    <label for="sidebar-checkbox" class="sidebar-toggle"></label>    


<!-- Toggleable sidebar -->
    <div class="sidebar" id="sidebar">
      <div class="sidebar-item">
        <p>Chris Wallace group:<br/>
        <b>statistics</b><br/>
        <em>applied to</em><br/> 
        <b>genomics</b><br/>
        <em>to understand</em><br/>
        <b>autoimmune diseases</b>.</p>
      </div>
      
      <nav class="sidebar-nav">
        <a class="sidebar-nav-item" href="index.html"><i class="fa fa-home"></i></a>
        <!-- <a href="./background.html">Background</a> |  -->
        <a class="sidebar-nav-item" href="./group.html">People</a>
        <a class="sidebar-nav-item" href="./research.html">Research</a>
        <a class="sidebar-nav-item" href="./data.html">Data</a>
        <a class="sidebar-nav-item" href="./code.html">Software</a>
        <!-- <a class="sidebar-nav-item" href="./vacancies.html">Vacancies</a> -->
        <!-- <a class="sidebar-nav-item" href="http://europepmc.org/search?query=AUTHORID:%220000-0001-9755-1703%22&sortby=Date">Publications</a> | -->
        <a class="sidebar-nav-item" href="./pubs2.html">Publications</a>
        <a class="sidebar-nav-item" href="./contact.html">Links and contact</a>
      </nav>
    </div>

<p>
This page holds information about datasets and code used in our papers.  Our intention
is to make the data and code we have used accessible.  If anything is
unclear, please <a href="https://github.com/chr1swallace/chr1swallace.github.io/issues">report it as an issue</a> so I (and you) can track it.  If
you really don't want to use github, as a last resort, email
<code>cew54</code> at <code>cam.ac.uk</code>.
</p>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1">Code and data organised by papers</h2>
<div class="outline-text-2" id="text-1">
</div><div id="outline-container-sec-1-1" class="outline-3">
<h3 id="sec-1-1">2015</h3>
<div class="outline-text-3" id="text-1-1">
</div><div id="outline-container-sec-1-1-1" class="outline-4">
<h4 id="sec-1-1-1"></h4>
<div class="outline-text-4" id="text-1-1-1">
<p>
<b>Paper</b>: Dissection of a Complex Disease Susceptibility Region Using a Bayesian Stochastic Search Approach to Fine Mapping. Wallace et al., <i>PLoS genetics</i> 2015.  [ <a href="http://ukpmc.ac.uk/abstract/MED/26106896"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1371/journal.pgen.1005272"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Code</b>: Uses the R package <a href="https://github.com/chr1swallace/GUESSFM">GUESSFM</a>.  For detailed pipeline of steps to run GUESSFM, see the <a href="https://github.com/chr1swallace/GUESSFM/tree/master/vignettes">vignettes</a>.
</p>

<p>
<b>Data</b>: <a href="#sec-2-1-5">T1D ImmunoChip Data</a> | <a href="#sec-2-1-3">MS ImmunoChip Data</a>
</p>
</div>
</div>



<div id="outline-container-sec-1-1-2" class="outline-4">
<h4 id="sec-1-1-2"></h4>
<div class="outline-text-4" id="text-1-1-2">
<p>
<b>Paper</b>: Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls. Fortune et al., <i>Nature Genetics</i> 2015.  [ <a href="http://ukpmc.ac.uk/abstract/MED/26053495"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ng.3330"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Data</b>: <a href="#sec-2-1-5">T1D ImmunoChip Data</a> | <a href="#sec-2-1-3">MS ImmunoChip Data</a> | <a href="#sec-2-1-4">RA ImmunoChip Data</a> | <a href="#sec-2-1-5">T1D ImmunoChip Data</a>
</p>

<p>
Complete results are available in the <a href="http://www.nature.com/ng/journal/v47/n7/full/ng.3330.html#supplementary-information">Supplementary Tables</a>.
</p>
</div>
</div>

<div id="outline-container-sec-1-1-3" class="outline-4">
<h4 id="sec-1-1-3"></h4>
<div class="outline-text-4" id="text-1-1-3">
<p>
<b>Paper</b>: Widespread seasonal gene expression reveals annual differences in human immunity and physiology. Dopico et al. <i>Nature Communications</i> 2015  [ <a href="http://ukpmc.ac.uk/abstract/MED/25965853"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ncomms8000"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Data</b>: <a href="#sec-2-2-3">BABYDIET gene expression</a> | <a href="#sec-2-2-4">T1D gene expression</a> | <a href="#sec-2-2-5">Asthma gene expression</a>
</p>
</div>
</div>

<div id="outline-container-sec-1-1-4" class="outline-4">
<h4 id="sec-1-1-4"></h4>
<div class="outline-text-4" id="text-1-1-4">
<p>
<b>Paper</b>: Fine mapping of type 1 diabetes susceptibility loci and evidence for colocalization of causal variants with lymphoid gene enhancers. Onengut-Gumuscu et al., <i>Nature Genetics</i> 2015 [ <a href="http://ukpmc.ac.uk/abstract/MED/25751624"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ng.3245"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Code</b>: R code used to perform
  numerical calculations exploring the appropriate p value thresholds
  for calling genetic association in one disease given genomewide
  significant genetic association at the same SNP in a related
  disease. <a href="https://github.com/chr1swallace/condpp">https://github.com/chr1swallace/condpp</a>.  
</p>

<p>
<b>Results</b>: Detailed results are curated at the <a href="http://www.immunobase.org/poster/type-1-diabetes-immunochip-study-onengut-gumuscu/">ImmunoBase poster page</a>, including interactive figures, T1D regions studied and identified, and a full list of the T1D credible SNP set.
</p>

<p>
<b>Data</b>: <a href="#sec-2-1-5">T1D ImmunoChip Data</a>
</p>
</div>
</div>

<div id="outline-container-sec-1-1-5" class="outline-4">
<h4 id="sec-1-1-5"></h4>
<div class="outline-text-4" id="text-1-1-5">
<p>
<b>Paper</b>: Integration of disease association and eQTL data using a Bayesian colocalisation approach highlights six candidate causal genes in immune-mediated diseases. Guo et al., <i>Human Molecular Genetics</i> 2015.  [ <a href="http://ukpmc.ac.uk/abstract/MED/25743184"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1093/hmg/ddv077"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Code</b>: Uses the R package <a href="https://github.com/chr1swallace/coloc/">coloc</a> (available also from <a href="http://cran.r-project.org/web/packages/coloc/index.html">CRAN</a>).
</p>

<p>
<b>Data</b>: <a href="#sec-2-1-7">ImmunoChip summary statistics</a> | <a href="#sec-2-2-1">Fairfax eQTL data</a> | <a href="#sec-2-2-6">Cardiogenics eQTL data</a>
</p>
</div>
</div>

<div id="outline-container-sec-1-1-6" class="outline-4">
<h4 id="sec-1-1-6"></h4>
<div class="outline-text-4" id="text-1-1-6">
<p>
<b>Paper</b>: Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis.  Bowes et al. <i>Nature Communications</i> 2015. [ <a href="http://www.nature.com/ncomms/2015/150205/ncomms7046/abs/ncomms7046.html"> <code>doi</code> </a> ] <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Code</b>: R script for fine mapping <a href="https://github.com/chr1swallace/finemap-psa">https://github.com/chr1swallace/finemap-psa</a> 
</p>

<p>
<b>Data</b>: RA p values required for input were downloaded from <a href="http://www.immunobase.org">ImmunoBase</a>.
</p>
</div>
</div>

<div id="outline-container-sec-1-1-7" class="outline-4">
<h4 id="sec-1-1-7"></h4>
<div class="outline-text-4" id="text-1-1-7">
<p>
<b>Paper</b>: A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics. Liley and Wallace, <i>PLoS Genetics</i> 2015. [ <a href="http://ukpmc.ac.uk/abstract/MED/25658688"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1371/journal.pgen.1004926"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Code</b>: James' code is on <a href="https://github.com/jamesliley/cFDR-common-controls">https://github.com/jamesliley/cFDR-common-controls</a>.
</p>

<p>
<b>Data</b>: GWAS summary statistics (p values) were downloaded from <i>Immunobase</i>. Also uses <a href="#sec-2-1-5">T1D ImmunoChip Data</a> for simulations to examine properties of the method.
</p>
</div>
</div>
</div>

<div id="outline-container-sec-1-2" class="outline-3">
<h3 id="sec-1-2">2014</h3>
<div class="outline-text-3" id="text-1-2">
</div><div id="outline-container-sec-1-2-1" class="outline-4">
<h4 id="sec-1-2-1"></h4>
<div class="outline-text-4" id="text-1-2-1">
<p>
<b>Paper</b>: Burren et al. VSEAMS: A pipeline for variant set enrichment analysis using summary GWAS data identifies IKZF3, BATF and ESRRA as key transcription factors in type 1 diabetes. <i>Bioinformatics</i> 2014.[ <a href="http://ukpmc.ac.uk/abstract/MED/25170024"> <code>ukpmc</code></a> ] <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
<b>Code</b>: Olly's pipeline is at <a href="http://github.com/ollyburren/vseams">http://github.com/ollyburren/vseams</a>.
</p>

<p>
<b>Data</b>: We used data from <i>Cusanovich et al</i>'s transcription factor knockdowns.  In fact, I think our paper was submitted only a month after theirs was published, because the preprint was shared on arXiv (and indexed on <a href="http://haldanessieve.org/2013/10/22/the-functional-consequences-of-variation-in-transcription-factor-binding">Haldane's Sieve</a>, which is where we saw it), and the authors readily shared their data pre-publication.  Open science working at its best.
</p>

<p>
We also used <a href="#sec-2-1-6">T1D GWAS Data</a>.
</p>
</div>
</div>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2">Datasets we have used</h2>
<div class="outline-text-2" id="text-2">
</div><div id="outline-container-sec-2-1" class="outline-3">
<h3 id="sec-2-1">GWAS</h3>
<div class="outline-text-3" id="text-2-1">
</div><div id="outline-container-sec-2-1-1" class="outline-4">
<h4 id="sec-2-1-1">ATD ImmunoChip Data</h4>
<div class="outline-text-4" id="text-2-1-1">
<p>
Need to be requested from the original study authors <a href="https://www.ncbi.nlm.nih.gov/pubmed/22922229">https://www.ncbi.nlm.nih.gov/pubmed/22922229</a>
</p>
</div>
</div>

<div id="outline-container-sec-2-1-2" class="outline-4">
<h4 id="sec-2-1-2">Coeliac ImmunoChip Data</h4>
<div class="outline-text-4" id="text-2-1-2">
<p>
Primary analysis is by <a href="http://www.nature.com/ng/journal/v43/n12/full/ng.998.html">Trynka et al</a>.  Genotype data  is hosted by the European Bioinformatics Institute, under accession number <a href="http://www.ebi.ac.uk/ega/studies/EGAS00000000053">EGAS00000000053</a>.
</p>
</div>
</div>

<div id="outline-container-sec-2-1-3" class="outline-4">
<h4 id="sec-2-1-3">MS ImmunoChip Data</h4>
<div class="outline-text-4" id="text-2-1-3">
<p>
MS ImmunoChip data was accessed through application to the <i>International Multiple Sclerosis Genetic Consortium (IMSGC)</i>.  Primary analysis of these data is presented in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882906/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882906/</a> and the dbGaP accession number for the  US cohort is phs000357.v2.p2.
</p>
</div>
</div>

<div id="outline-container-sec-2-1-4" class="outline-4">
<h4 id="sec-2-1-4">RA ImmunoChip Data</h4>
<div class="outline-text-4" id="text-2-1-4">
<p>
Primary analysis was by <a href="http://www.nature.com/ng/journal/v44/n12/full/ng.2462.html">Eyre et al.</a> and data were accessed through a request to the authors.
</p>
</div>
</div>

<div id="outline-container-sec-2-1-5" class="outline-4">
<h4 id="sec-2-1-5">T1D ImmunoChip Data</h4>
<div class="outline-text-4" id="text-2-1-5">
<p>
is available from dbGaP (<a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2</a>)
</p>
</div>
</div>

<div id="outline-container-sec-2-1-6" class="outline-4">
<h4 id="sec-2-1-6">T1D GWAS Data</h4>
<div class="outline-text-4" id="text-2-1-6">
<p>
2000 T1D samples were genotyped as part of the WTCCC (and controls) - data access  is described at <a href="https://www.wtccc.org.uk/info/access_to_data_samples.html">https://www.wtccc.org.uk/info/access_to_data_samples.html</a>. An additional 4000 cases were genotyped by the T1DGC, available at <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2</a>
</p>

<p>
Autoantibody data on T1D subjects is available by contacting study authors from <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002216">http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002216</a>.
</p>
</div>
</div>

<div id="outline-container-sec-2-1-7" class="outline-4">
<h4 id="sec-2-1-7">ImmunoChip summary statistics</h4>
<div class="outline-text-4" id="text-2-1-7">
<p>
<b>ImmunoBase</b>: ImmunoChip summary GWAS statistics from multiple studies <b>used to be</b> nicely curated and available from <a href="http://www.immunobase.org">ImmunoBase</a> if you first register (free) then go to <code>Data..downloads</code> on the right, then click through to <code>data/immunochip/association_stats</code>.  I'm not sure how to access it now.
</p>
</div>
</div>
</div>

<div id="outline-container-sec-2-2" class="outline-3">
<h3 id="sec-2-2">Expression data</h3>
<div class="outline-text-3" id="text-2-2">
</div><div id="outline-container-sec-2-2-1" class="outline-4">
<h4 id="sec-2-2-1">Fairfax eQTL data</h4>
<div class="outline-text-4" id="text-2-2-1">
<p>
Ben Fairfax has published eQTL papers on sorted cells (monocytes, stimulated monocytes, B cells) from close to 300 samples.  The expression data are freely available, but the genotype data require a data access agreement, available as detailed in either paper.
</p>

<p>
<i>Fairfax et al. (2012) Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502-510</i>
</p>

<p>
gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>.
</p>

<p>
<i>Fairfax et al. (2014) Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science 343, 1246949</i>
</p>

<p>
gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>, <a href="http://www.ebi.ac.uk/ega/EGAD00010000520">http://www.ebi.ac.uk/ega/EGAD00010000520</a>.
</p>
</div>
</div>

<div id="outline-container-sec-2-2-2" class="outline-4">
<h4 id="sec-2-2-2">Cusanovich et al's transcription factor knockdown</h4>
<div class="outline-text-4" id="text-2-2-2">
<p>
See <a href="http://europepmc.org/abstract/MED/24603674">http://europepmc.org/abstract/MED/24603674</a> for the paper.  This
dataset has a special place in my heart, as we requested it after
Cusanovich posted a preprint and it was shared quickly and freely.  As
a result, I <i>think</i> our paper using this data was actually preprinted and submitted
before the Cusanovich paper was formally published in a journal.  A
true example of preprint publishing furthering research.
</p>
</div>
</div>

<div id="outline-container-sec-2-2-3" class="outline-4">
<h4 id="sec-2-2-3">BABYDIET gene expression</h4>
<div class="outline-text-4" id="text-2-2-3">
<p>
PBMC gene expression from longitudinally sampled children at risk of type 1 diabetes, described by <a href="http://www.ncbi.nlm.nih.gov/pubmed/24561305">Ferreira et al. (2014)</a>.
<a href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a>, accession no. E-MTAB-1724
</p>
</div>
</div>

<div id="outline-container-sec-2-2-4" class="outline-4">
<h4 id="sec-2-2-4">T1D gene expression</h4>
<div class="outline-text-4" id="text-2-2-4">
<p>
The T1D PBMCs were collected as part of the Genetic Resource
Investigating Diabetes (GRID) cohort collection
(<a href="http://www.childhood-diabetes.org.uk/grid.shtml">http://www.childhood-diabetes.org.uk/grid.shtml</a>).  Expression data at
<a href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a>, accession no. E-MTAB-1724
</p>
</div>
</div>

<div id="outline-container-sec-2-2-5" class="outline-4">
<h4 id="sec-2-2-5">Asthma gene expression</h4>
<div class="outline-text-4" id="text-2-2-5">
<p>
Asthma cohort described by <a href="http://www.ncbi.nlm.nih.gov/pubmed/21779351">Bjornsdottir et al. (2011)</a>. Raw and normalized data are deposited with ArrayExpress ( <a href="http://www.ebi.ac.uk/arrayexpress/">http://www.ebi.ac.uk/arrayexpress/</a>, E-GEOD-19301).
</p>
</div>
</div>


<div id="outline-container-sec-2-2-6" class="outline-4">
<h4 id="sec-2-2-6">Cardiogenics eQTL data</h4>
<div class="outline-text-4" id="text-2-2-6">
<p>
<i>Garnier S et al. (2013) Genome-wide haplotype analysis of cis expression quantitative trait Loci in monocytes. <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003240">PLoS Genet. 9(1):e1003240</a>.</i>
</p>

<p>
Monocytes eQTL study available from EGA: <a href="https://www.ebi.ac.uk/ega/studies/EGAS00001000411">https://www.ebi.ac.uk/ega/studies/EGAS00001000411</a>
</p>
</div>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="date">Created: 2017-01-25 Wed 21:12</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
