<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>Statistics, genomics and autoimmune diseases</title>
<!-- 2015-02-20 Fri 08:07 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<meta  name="author" content="chris" />
<link rel="stylesheet" type="text/css" href="css/mystyle.css">
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">Statistics, genomics and autoimmune diseases</h1>
<p>
This page holds information about datasets and code used in
statistical genetics papers from Chris Wallace's group.  Our intention
is to make the data and code we have used accessible.  If anything is
unclear, please <a href="https://github.com/chr1swallace/chr1swallace.github.io/issues">report it as an issue</a> so I (and you) can track it.  If
you really don't want to use github, as a last resort, email
<code>chris.wallace</code> at <code>cimr.cam.ac.uk</code>.
</p>

<p>
Information that exists here falls under:
</p>

<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#sec-1">Code and data organised by papers</a>
<ul>
<li><a href="#sec-1-1">2015</a>
<ul>
<li><a href="#sec-1-1-1">Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" bioRxiv preprint.</a></li>
<li><a href="#sec-1-1-2">Bowes et al "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis" Nat Commun.</a></li>
<li><a href="#sec-1-1-3">Liley and Wallace "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics" PLOS Genet.</a></li>
</ul>
</li>
<li><a href="#sec-1-2">2014</a>
<ul>
<li><a href="#sec-1-2-1">Burren et al "VSEAMS: a pipeline for variant set enrichment analysis using summary GWAS data identifies IKZF3, BATF and ESRRA as key transcription factors in type 1 diabetes" Bioinformatics.</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#sec-2">Datasets we use</a>
<ul>
<li><a href="#sec-2-1">GWAS</a>
<ul>
<li><a href="#sec-2-1-1">T1D ImmunoChip Data</a></li>
<li><a href="#sec-2-1-2">T1D GWAS Data</a></li>
<li><a href="#sec-2-1-3">ImmunoBase</a></li>
</ul>
</li>
<li><a href="#sec-2-2">Expression data</a>
<ul>
<li><a href="#sec-2-2-1">Fairfax eQTL data</a></li>
<li><a href="#sec-2-2-2">Cusanovich et al's transcription factor knockdown</a></li>
<li><a href="#sec-2-2-3">BABYDIET</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#sec-3">Larger repositories/software packages</a></li>
</ul>
</div>
</div>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1">Code and data organised by papers</h2>
<div class="outline-text-2" id="text-1">
</div><div id="outline-container-sec-1-1" class="outline-3">
<h3 id="sec-1-1">2015</h3>
<div class="outline-text-3" id="text-1-1">
</div>

<div id="outline-container-sec-1-1-1" class="outline-4">
<h4 id="sec-1-1-1">Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" <a href="http://biorxiv.org/content/early/2015/02/12/015164">bioRxiv preprint</a>.</h4>
<div class="outline-text-4" id="text-1-1-1">
<p>
Uses the R package <a href="https://github.com/chr1swallace/GUESSFM">GUESSFM</a>.  For detailed pipeline of steps to run GUESSFM, see the <a href="https://github.com/chr1swallace/GUESSFM/tree/master/vignettes">vignettes</a>.
</p>

<p>
<a href="#sec-2-1-1">T1D ImmunoChip Data</a>
</p>

<p>
MS data was accessed through application to the <i>International Multiple Sclerosis Genetic Consortium (IMSGC)</i>.
</p>
</div>
</div>

<div id="outline-container-sec-1-1-2" class="outline-4">
<h4 id="sec-1-1-2">Bowes et al "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis" <a href="http://www.nature.com/ncomms/2015/150205/ncomms7046/abs/ncomms7046.html">Nat Commun</a>.</h4>
<div class="outline-text-4" id="text-1-1-2">
<p>
R script for fine mapping <a href="https://github.com/chr1swallace/finemap-psa">https://github.com/chr1swallace/finemap-psa</a> 
RA p values required for input were downloaded from <a href="http://www.immunobase.org">ImmunoBase</a>.
</p>
</div>
</div>

<div id="outline-container-sec-1-1-3" class="outline-4">
<h4 id="sec-1-1-3">Liley and Wallace "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics" <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004926">PLOS Genet</a>.</h4>
<div class="outline-text-4" id="text-1-1-3">
<p>
Data Availability: GWAS summary statistics (p values) were downloaded from <i>Immunobase</i>. Also uses <a href="#sec-2-1-1">T1D ImmunoChip Data</a> for simulations to examine properties of the method.
</p>

<p>
James' code is on <a href="https://github.com/jamesliley/cFDR-common-controls">https://github.com/jamesliley/cFDR-common-controls</a>.
</p>
</div>
</div>
</div>
<div id="outline-container-sec-1-2" class="outline-3">
<h3 id="sec-1-2">2014</h3>
<div class="outline-text-3" id="text-1-2">
</div><div id="outline-container-sec-1-2-1" class="outline-4">
<h4 id="sec-1-2-1">Burren et al "VSEAMS: a pipeline for variant set enrichment analysis using summary GWAS data identifies IKZF3, BATF and ESRRA as key transcription factors in type 1 diabetes" <a href="http://europepmc.org/abstract/MED/25170024">Bioinformatics</a>.</h4>
<div class="outline-text-4" id="text-1-2-1">
<p>
Olly's pipeline is at <a href="http://github.com/ollyburren/vseams">http://github.com/ollyburren/vseams</a>.
</p>

<p>
We used data from <i>Cusanovich et al</i>'s transcription factor knockdowns.  In fact, I think our paper was submitted only a month after theirs was published, because the preprint was shared on arXiv (and indexed on <a href="http://haldanessieve.org/2013/10/22/the-functional-consequences-of-variation-in-transcription-factor-binding">Haldane's Sieve</a>, which is where we saw it), and the authors readily shared their data pre-publication.  Open science working at its best.
</p>

<p>
We also used <a href="#sec-2-1-2">T1D GWAS Data</a>.
</p>
</div>
</div>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2">Datasets we use</h2>
<div class="outline-text-2" id="text-2">
</div><div id="outline-container-sec-2-1" class="outline-3">
<h3 id="sec-2-1">GWAS</h3>
<div class="outline-text-3" id="text-2-1">
</div><div id="outline-container-sec-2-1-1" class="outline-4">
<h4 id="sec-2-1-1">T1D ImmunoChip Data</h4>
<div class="outline-text-4" id="text-2-1-1">
<p>
from dbGaP (<a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v2.p2">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v2.p2</a>)
</p>
</div>
</div>

<div id="outline-container-sec-2-1-2" class="outline-4">
<h4 id="sec-2-1-2">T1D GWAS Data</h4>
<div class="outline-text-4" id="text-2-1-2">
<p>
There have been (at least) three T1D GWAS studies.  We used all three, the details of how to access these data are best found on <a href="http://www.t1dbase.org/poster/accessing-external-private-genotyping-data/">T1DBase</a>, and the publications they come from are summarised on the <a href="https://www-gene.cimr.cam.ac.uk/publications/human_data.shtml#gwas">DIL's</a> website.  
</p>
</div>
</div>

<div id="outline-container-sec-2-1-3" class="outline-4">
<h4 id="sec-2-1-3">ImmunoBase</h4>
<div class="outline-text-4" id="text-2-1-3">
<p>
ImmunoChip summary GWAS statistics from multiple studies are nicely curated and available from <a href="http://www.immunobase.org">ImmunoBase</a> if you first register (free) then go to <code>Data..downloads</code> on the right, then click through to <code>data/immunochip/association_stats</code>.
</p>
</div>
</div>
</div>

<div id="outline-container-sec-2-2" class="outline-3">
<h3 id="sec-2-2">Expression data</h3>
<div class="outline-text-3" id="text-2-2">
</div><div id="outline-container-sec-2-2-1" class="outline-4">
<h4 id="sec-2-2-1">Fairfax eQTL data</h4>
<div class="outline-text-4" id="text-2-2-1">
<p>
Ben Fairfax has published eQTL papers on sorted cells (monocytes, stimulated monocytes, B cells) from close to 300 samples.  The expression data are freely available, but the genotype data require a data access agreement, available as detailed in either paper.
</p>

<p>
<i>Fairfax et al. (2012) Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502-510</i>
</p>

<p>
gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>.
</p>

<p>
<i>Fairfax et al. (2014) Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science 343, 1246949</i>
</p>

<p>
gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>, <a href="http://www.ebi.ac.uk/ega/EGAD00010000520">http://www.ebi.ac.uk/ega/EGAD00010000520</a>.
</p>
</div>
</div>

<div id="outline-container-sec-2-2-2" class="outline-4">
<h4 id="sec-2-2-2">Cusanovich et al's transcription factor knockdown</h4>
<div class="outline-text-4" id="text-2-2-2">
<p>
See <a href="http://europepmc.org/abstract/MED/24603674">http://europepmc.org/abstract/MED/24603674</a> for the paper.  
</p>
</div>
</div>

<div id="outline-container-sec-2-2-3" class="outline-4">
<h4 id="sec-2-2-3">BABYDIET</h4>
<div class="outline-text-4" id="text-2-2-3">
<p>
PBMC gene expression from longitudinally sampled children at risk of type 1 diabetes
<a href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a>, accession no. E-MTAB-1724
</p>
</div>
</div>
</div>
</div>

<div id="outline-container-sec-3" class="outline-2">
<h2 id="sec-3">Larger repositories/software packages</h2>
<div class="outline-text-2" id="text-3">
<p>
See <a href="https://github.com/chr1swallace?tab=repositories">https://github.com/chr1swallace?tab=repositories</a> for everything else
</p>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: chris</p>
<p class="date">Created: 2015-02-20 Fri 08:07</p>
<p class="creator"><a href="http://www.gnu.org/software/emacs/">Emacs</a> 24.3.1 (<a href="http://orgmode.org">Org</a> mode 8.2.10)</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
