#+TITLE: Chris Wallace: data
#+SETUPFILE: ./setup.org

#+INCLUDE: ./nav.org

# <script type="text/javascript">
  # function show(id) {
  #   document.getElementById(id).style.visibility = "visible";
  # }
  # function hide(id) {
  #   document.getElementById(id).style.visibility = "hidden";
  # }
# </script>

This page holds information about datasets and code used in
statistical genetics papers from Chris Wallace's group.  Our intention
is to make the data and code we have used accessible.  If anything is
unclear, please [[https://github.com/chr1swallace/chr1swallace.github.io/issues][report it as an issue]] so I (and you) can track it.  If
you really don't want to use github, as a last resort, email
=chris.wallace= at =cimr.cam.ac.uk=.

Information that exists here falls under:

* Code and data organised by papers

** 2015

*** Onengut-Gumuscu et al "Fine mapping of type 1 diabetes susceptibility loci and evidence for colocalization of causal variants with lymphoid gene enhancers"

*Paper*: [[http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3245.html][Nat Genet]] | [[http://www-gene.cimr.cam.ac.uk/staff/wallace/ichip-preprint.pdf][preprint]]

*Code*: R code used to perform
  numerical calculations exploring the appropriate p value thresholds
  for calling genetic association in one disease given genomewide
  significant genetic association at the same SNP in a related
  disease. https://github.com/chr1swallace/condpp.  

*Results*: Detailed results are curated at the [[http://www.immunobase.org/poster/type-1-diabetes-immunochip-study-onengut-gumuscu/][ImmunoBase poster page]], including interactive figures, T1D regions studied and identified, and a full list of the T1D credible SNP set.

*Data*: [[T1D ImmunoChip Data]]

*** Guo et al "Integration of disease association and eQTL data using a Bayesian colocalisation approach highlights six candidate causal genes in immune-mediated diseases"

*Paper*: [[http://dx.doi.org/10.1093/hmg/ddv077][Hum Mol Genet]]

*Code*: Uses the R package [[https://github.com/chr1swallace/coloc/][coloc]] (available also from [[http://cran.r-project.org/web/packages/coloc/index.html][CRAN]]).

*Data*: [[ImmunoChip summary statistics]] | [[Fairfax eQTL data]] | [[Cardiogenics eQTL data]]

*** Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping"

*Paper*: [[http://biorxiv.org/content/early/2015/02/12/015164][bioRxiv preprint]].

*Code*: Uses the R package [[https://github.com/chr1swallace/GUESSFM][GUESSFM]].  For detailed pipeline of steps to run GUESSFM, see the [[https://github.com/chr1swallace/GUESSFM/tree/master/vignettes][vignettes]].

*Data*: [[T1D ImmunoChip Data]]

MS data was accessed through application to the [[International Multiple Sclerosis Genetic Consortium][International Multiple Sclerosis Genetic Consortium (IMSGC)]].

*** Bowes et al "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis"
*Paper*: [[http://www.nature.com/ncomms/2015/150205/ncomms7046/abs/ncomms7046.html][Nat Commun]].

*Code*: R script for fine mapping https://github.com/chr1swallace/finemap-psa 

*Data*: RA p values required for input were downloaded from [[http://www.immunobase.org][ImmunoBase]].

*** Liley and Wallace "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics"
*Paper*: [[http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004926][PLOS Genet]].

*Code*: James' code is on https://github.com/jamesliley/cFDR-common-controls.

*Data*: GWAS summary statistics (p values) were downloaded from [[Immunobase]]. Also uses [[T1D ImmunoChip Data]] for simulations to examine properties of the method.

** 2014
*** Burren et al "VSEAMS: a pipeline for variant set enrichment analysis using summary GWAS data identifies IKZF3, BATF and ESRRA as key transcription factors in type 1 diabetes"
*Paper*: [[http://europepmc.org/abstract/MED/25170024][Bioinformatics]].

*Code*: Olly's pipeline is at [[http://github.com/ollyburren/vseams]].

*Data*: We used data from [[Cusanovich][Cusanovich et al]]'s transcription factor knockdowns.  In fact, I think our paper was submitted only a month after theirs was published, because the preprint was shared on arXiv (and indexed on [[http://haldanessieve.org/2013/10/22/the-functional-consequences-of-variation-in-transcription-factor-binding][Haldane's Sieve]], which is where we saw it), and the authors readily shared their data pre-publication.  Open science working at its best.

We also used [[T1D GWAS Data]].

* Datasets we have used

** GWAS
*** T1D ImmunoChip Data 

from dbGaP ([[http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v2.p2]])

*** T1D GWAS Data

There have been (at least) three T1D GWAS studies.  We used all three, the details of how to access these data are best found on [[http://www.t1dbase.org/poster/accessing-external-private-genotyping-data/][T1DBase]], and the publications they come from are summarised on the [[https://www-gene.cimr.cam.ac.uk/publications/human_data.shtml#gwas][DIL's]] website.  

*** ImmunoChip summary statistics

*ImmunoBase*: ImmunoChip summary GWAS statistics from multiple studies are nicely curated and available from [[http://www.immunobase.org][ImmunoBase]] if you first register (free) then go to =Data..downloads= on the right, then click through to =data/immunochip/association_stats=.

** Expression data

*** Fairfax eQTL data 

Ben Fairfax has published eQTL papers on sorted cells (monocytes, stimulated monocytes, B cells) from close to 300 samples.  The expression data are freely available, but the genotype data require a data access agreement, available as detailed in either paper.

/Fairfax et al. (2012) Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502-510/

gene expression data: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945; genotyping data: http://www.ebi.ac.uk/ega/EGAD00010000144.

/Fairfax et al. (2014) Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science 343, 1246949/

gene expression data: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232; genotyping data: http://www.ebi.ac.uk/ega/EGAD00010000144, http://www.ebi.ac.uk/ega/EGAD00010000520.

*** Cusanovich et al's transcription factor knockdown 

See [[http://europepmc.org/abstract/MED/24603674]] for the paper.  This
dataset has a special place in my heart, as we requested it after
Cusanovich posted a preprint and it was shared quickly and freely.  As
a result, I /think/ our paper using this data was actually submitted
before the Cusanovich paper was formally published in a journal.  A
true example of preprint publishing furthering research.

*** BABYDIET 

PBMC gene expression from longitudinally sampled children at risk of type 1 diabetes
http://www.ebi.ac.uk/arrayexpress, accession no. E-MTAB-1724

*** Cardiogenics eQTL data

/Garnier S et al. (2013) Genome-wide haplotype analysis of cis expression quantitative trait Loci in monocytes. [[http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003240][PLoS Genet. 9(1):e1003240]]./

Monocytes eQTL study available from EGA: [[https://www.ebi.ac.uk/ega/studies/EGAS00001000411]]



* Larger repositories/software packages

See [[https://github.com/chr1swallace?tab=repositories]] for everything else
