<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>People</title>
<!-- 2017-09-12 Tue 09:25 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center; }
  .todo   { font-family: monospace; color: red; }
  .done   { color: green; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  pre.src-sh:before    { content: 'sh'; }
  pre.src-bash:before  { content: 'sh'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-R:before     { content: 'R'; }
  pre.src-perl:before  { content: 'Perl'; }
  pre.src-java:before  { content: 'Java'; }
  pre.src-sql:before   { content: 'SQL'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.right  { text-align: center;  }
  th.left   { text-align: center;   }
  th.center { text-align: center; }
  td.right  { text-align: right;  }
  td.left   { text-align: left;   }
  td.center { text-align: center; }
  dt { font-weight: bold; }
  .footpara:nth-child(2) { display: inline; }
  .footpara { display: block; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  /*]]>*/-->
</style>
<link rel="stylesheet" type="text/css" href="css/poole.css"/>
<link rel="stylesheet" type="text/css" href="css/lanyon.css"/>
<link rel="stylesheet" type="text/css" href="css/tabs.css"/>
<link rel="stylesheet" type="text/css" href="css/colours.css"/>
<script type="text/javascript" src="styles/bigblow/js/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery-ui-1.10.2.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.localscroll-min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.scrollTo-1.4.3.1-min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.zclip.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/bigblow.js"></script>
<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">People</h1>
    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior.  -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">
    <label for="sidebar-checkbox" class="sidebar-toggle"></label>    


<!-- Toggleable sidebar -->
    <div class="sidebar" id="sidebar">
      <div class="sidebar-item">
        <p>Chris Wallace group:<br/>
        <b>statistics</b><br/>
        <em>applied to</em><br/> 
        <b>genomics</b><br/>
        <em>to understand</em><br/>
        <b>autoimmune diseases</b>.</p>
      </div>
      
      <nav class="sidebar-nav">
        <a class="sidebar-nav-item" href="index.html"><i class="fa fa-home"></i></a>
        <!-- <a href="./background.html">Background</a> |  -->
        <a class="sidebar-nav-item" href="./group.html">People</a>
        <a class="sidebar-nav-item" href="./research.html">Research</a>
        <a class="sidebar-nav-item" href="./data.html">Data</a>
        <a class="sidebar-nav-item" href="./code.html">Software</a>
        <!-- <a class="sidebar-nav-item" href="./vacancies.html">Vacancies</a> -->
        <!-- <a class="sidebar-nav-item" href="http://europepmc.org/search?query=AUTHORID:%220000-0001-9755-1703%22&sortby=Date">Publications</a> | -->
        <a class="sidebar-nav-item" href="./pubs2.html">Publications</a>
        <a class="sidebar-nav-item" href="./talks.html">Presentations</a>
        <a class="sidebar-nav-item" href="./contact.html">Links and contact</a>
      </nav>
    </div>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1">Current members</h2>
<div class="outline-text-2" id="text-1">
</div><div id="outline-container-sec-1-1" class="outline-3">
<h3 id="sec-1-1">Jennifer Asimit</h3>
<div class="outline-text-3" id="text-1-1">
<img src="pics/Jennifer_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Jenn Asimit is a senior investigator statistician with an interest in statistical methodology development for the analysis of multiple traits, as well as fine-mapping and rare variant association approaches. She recently held a MRC Methodology Research fellowship at the Sanger institute and has developed several methods and software for rare variant association analyses and the overlap analysis of traits.
</p>
</div>
</div>

<div id="outline-container-sec-1-2" class="outline-3">
<h3 id="sec-1-2">Oliver Burren</h3>
<div class="outline-text-3" id="text-1-2">
<img src="pics/Olly_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
<a href="http://ollyburren.github.io">Olly Burren</a> is undertaking a PhD, researching methods to integrate genomic and genetic datasets to better understand autoimmune disease mechanisms. He was previously head of Genome Informatics in the <a href="https://www-gene.cimr.cam.ac.uk/">Diabetes and Inflammation Laboratory</a> where he was responsible for the creation and smooth running of the <a href="http://www.immunobase.org">ImmunoBase</a>,  <a href="http://www.t1dbase.org">T1DBase</a> and <a href="http://www.chicp.org">CHiCP</a> web resources which collate data and summary information from genomic, GWAS and ImmunoChip analyses of autoimmune diseases.  He continues to maintain  <a href="http://www.chicp.org">CHiCP</a> as a means of visualizing Capture Hi-C datasets.
</p>
</div>
</div>

<div id="outline-container-sec-1-3" class="outline-3">
<h3 id="sec-1-3">Alessandra Cabassi</h3>
<div class="outline-text-3" id="text-1-3">
<img src="pics/alessandra_150.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Alessandra Cabassi is a PhD student at the MRC Biostatistics Unit, jointly supervised by <a href="http://www.mrc-bsu.cam.ac.uk/people/in-alphabetical-order/h-to-m/paul-kirk/">Paul Kirk</a>. She is working on Bayesian nonparametric approaches for integrative modelling of genomic "big data".
</p>
</div>
</div>
<div id="outline-container-sec-1-4" class="outline-3">
<h3 id="sec-1-4">Nastasiya Grinberg</h3>
<div class="outline-text-3" id="text-1-4">
<img src="pics/nfg_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Nastasiya Grinberg is a Wellcome Trust funded postdoctoral researcher. Her research interests revolve around applications of machine learning and data mining techniques to GWAS data and in bioinformatics. She also has a special interest in scientific computing.
</p>
</div>
</div>

<div id="outline-container-sec-1-5" class="outline-3">
<h3 id="sec-1-5">James Liley</h3>
<div class="outline-text-3" id="text-1-5">
<img src="pics/jamesliley_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
James Liley is a NIHR/BRC funded student on the Wellcome Trust four-year PhD programme in Mathematical Genomics and Medicine. He is working on analysis of GWAS and similar data across multiple diseases and in disease subtypes.
</p>
</div>
</div>


<div id="outline-container-sec-1-6" class="outline-3">
<h3 id="sec-1-6">Chris Wallace</h3>
<div class="outline-text-3" id="text-1-6">
<img src="pic_200.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
I am a statistician who works with biological datasets to understand
the mechanisms underlying human disease and identify possible
treatments.  Such work is only possible with close collaboration with
biologists, and I moved to the <a href="http://www.cimr.cam.ac.uk">Cambridge
Institute for Medical Research</a> in 2009 to pursue these
collaborations, most closely with the <a href="http://www-gene.cimr.cam.ac.uk">Diabetes and Inflammation
Laboratory</a>.  During
the GWAS era, we, and others, identified many genetic polymorphisms
that alter risk of human diseases such as type 1 diabetes and
hypertension.  My focus has now mostly shifted towards understanding
the cell specific mechanism through which these variants affect risk
of different immune mediated diseases and intermediate biological
traits such as gene expression.
</p>

<p>
Statistically, my current work is focused on variable
selection, Bayesian model averaging and empirical Bayes false
discovery rates, applied to integrated analysis of multiple related
datasets.
</p>

<p>
In Feburary 2016 I moved to the <a href="http://www.med.cam.ac.uk/">Department of Medicine</a> as a PI with the aim of establishing new collaborations.  
I am funded by the <a href="http://www.wellcome.ac.uk/">Wellcome Trust</a> as a Senior Research Fellow  and hold an honorary Programme Leader  position  at the <a href="http://www.mrc-bsu.cam.ac.uk/">MRC Biostatistics Unit</a>.  My aim is to form a group that mixes detailed biological understanding with excellent statistical inference to make a meaningful contribution to the understanding of autoimmune diseases.
</p>

<p>
<b>Previously</b> 
</p>
<dl class="org-dl">
<dt> 2009-2016 </dt><dd>Wellcome Trust Career Development Fellow in the  <a href="http://www-gene.cimr.cam.ac.uk">Diabetes and Inflammation Laboratory</a> (and head of the DIL stats group from 2012)
</dd>
<dt> 2003-2008 </dt><dd>Research Fellow working with          the <a href="http://www.brightstudy.ac.uk/">BRIGHT</a> study into          the genetics of hypertension.
</dd>
<dt> 1999-2003 </dt><dd>PhD: <a href="http://figshare.com/articles/Human_genetic_susceptibility_to_leprosy_methodological_issues_in_a_linkage_analysis_of_extended_pedigrees_from_Karonga_district_Malawi/963086"><i>Genetic Susceptibility to leprosy: methodological issues in a linkage  analysis of extended pedigrees from Karonga District, Malawi</i></a>,          <a href="http://www.lshtm.ac.uk">LSHTM</a>, part of the          <a href="http://www.lshtm.ac.uk/eph/ide/research/kps">Karonga Prevention Study</a>. 
</dd>
</dl>
<dl class="org-dl">
<dt> 1997-1999 </dt><dd>Statistical programmer, <a href="http://www.imperial.ac.uk">Imperial</a>.
</dd>
</dl>
</div>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2">Visiting members</h2>
<div class="outline-text-2" id="text-2">
</div><div id="outline-container-sec-2-1" class="outline-3">
<h3 id="sec-2-1">Loes Rutten-Jacobs</h3>
<div class="outline-text-3" id="text-2-1">
<img src="pics/Loes_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
<a href="http://www.neuroscience.cam.ac.uk/directory/profile.php?lr406">Loes Rutten-Jacobs</a> is a British Heart Foundation Immediate Research Fellow in the <a href="http://www.neurology.cam.ac.uk/neurology-unit-research-groups/stroke-research-group/">Stroke Research Group</a> of the Department of Clinical Neurosciences, University of Cambridge and is visiting our group until December 2018. In her research she aims to identify novel pathophysiological mechanisms underlying cerebral small vessel disease, using genetic and epidemiologic approaches.        
</p>
</div>
</div>
</div>

<div id="outline-container-sec-3" class="outline-2">
<h2 id="sec-3">Alumni</h2>
<div class="outline-text-2" id="text-3">
</div><div id="outline-container-sec-3-1" class="outline-3">
<h3 id="sec-3-1">Chris Eijsbouts (2016-2017)</h3>
<div class="outline-text-3" id="text-3-1">
<img src="pics/ChrisE.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Chris Eijsbouts first joined the group as an MPhil student in Computational Biology for a 3 month summer project.  He stayed on afterwards as a research assistant, to continue his work on the interpretation of Capture Hi-C data. The goal of this project was to develop a method for improved identification of chromatin contacts which regulate gene expression, thereby facilitating the interpretation of GWAS studies.
He will start a PhD in Oxford in Autumn 2017.
</p>
</div>
</div>

<div id="outline-container-sec-3-2" class="outline-3">
<h3 id="sec-3-2">Steven Kiddle (2014-2017)</h3>
<div class="outline-text-3" id="text-3-2">
<img src="pics/kiddle_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
<a href="https://kclpure.kcl.ac.uk/portal/steven.kiddle.html">Steven Kiddle</a> held an MRC Career Development Fellow in Biostatistics based at King’s College London, on which Chris Wallace was a statistical mentor. He visited our group in Cambridge from 2015-6, working on clustering approaches for time series and biomarkers of Alzheimer’s Disease.
</p>

<p>
He now holds a new MRC Career Development Fellowship and is based in the  <a href="http://www.mrc-bsu.cam.ac.uk">MRC Biostatistics Unit</a>.
</p>
</div>
</div>



<div id="outline-container-sec-3-3" class="outline-3">
<h3 id="sec-3-3">Mary Fortune (2013-2017)</h3>
<div class="outline-text-3" id="text-3-3">
<img src="pics/mary_150.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
<a href="http://www.damtp.cam.ac.uk/people/mdf34/">Mary Fortune</a>  worked as a Wellcome Trust PhD student in the <a href="http://cato.medschl.cam.ac.uk/graduate-programmes/phd-programmes/wellcome-trust-mathematical-genomics-and-medicine-mgm/">Mathematical Genomics and Medicine</a> programme on colocalisation analysis in dependent datasets and methods for fitting models of genetic causality to summary GWAS data. She stayed on in the group to complete papers arising from her PhD work and has now moved to the <a href="http://www.mrc-bsu.cam.ac.uk">MRC Biostatistics Unit</a>.
</p>
</div>

<div id="outline-container-sec-3-3-1" class="outline-4">
<h4 id="sec-3-3-1">Key Publications</h4>
<div class="outline-text-4" id="text-3-3-1">
<p>
Thesis: <a href="https://doi.org/10.17863/CAM.10463">Statistical Techniques to Fine Map the Related Genetic Aetiology of Autoimmune Diseases</a>
</p>

<p>
Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls. Fortune et al. <i>Nature Genetics</i> 2015, vol. 47, no. 7, pp. 839-846.  [ <a href="http://ukpmc.ac.uk/abstract/MED/26053495"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ng.3330"><code>doi</code></a> ]
</p>
</div>
</div>
</div>



<div id="outline-container-sec-3-4" class="outline-3">
<h3 id="sec-3-4">Lucia Cilloni (Summer 2015)</h3>
<div class="outline-text-3" id="text-3-4">
<img src="pics/LuciaCilloni.png" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Lucia did her BSc in Mathematics at the University of Newcastle Upon
Tyne and her MPhil in Computational Biology at
Cambridge University. She did her summer project in the Wallace group on
eQTL-GWAS colocalisation analysis to identify causal genes for Type 1
Diabetes. She is now studying for a PhD in Epidemiology at
Imperial College London.
</p>
</div>
</div>

<div id="outline-container-sec-3-5" class="outline-3">
<h3 id="sec-3-5">Hui Guo (2010-2014)</h3>
<div class="outline-text-3" id="text-3-5">
<img src="pics/hui.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Hui worked as a post doctoral researcher on gene expression data, and led the analysis of our study relating gene expression in PBMCs from children at risk of type 1 diabetes to autoantibody positivity.  She is now a lecturer in biostatistics at <a href="http://www.population-health.manchester.ac.uk/staff/HuiGuo/">University of Manchester</a>.
</p>
</div>
</div>
<div id="outline-container-sec-3-6" class="outline-3">
<h3 id="sec-3-6">Xin Yang (2011-2014)</h3>
<div class="outline-text-3" id="text-3-6">
<img src="pics/xin.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
Xin's PhD focused on analysis of all kinds of next gen sequencing data - pooled DNA, RNA
seq, and bisulfite seq.  She had a special interest in modelling the extra binomial
variation induced by pooling.  Her thesis <a href="http://figshare.com/articles/Statistical_analysis_of_next_generation_sequencing_data_with_applications_to_type_1_diabetes_studies/1598221"><i>Statistical analysis of next-generation sequencing data with applications to type 1 diabetes studies</i></a> is on figshare.
</p>

<blockquote>
<p>
<b>Key paper</b>: Extra-binomial variation approach for analysis of pooled DNA sequencing data. Yang et al. <i>Bioinformatics</i> 2012, vol. 28, no. 22, pp. 2898-2904. [ <a href="http://dx.doi.org/10.1093/bioinformatics/bts553"><code>doi</code></a> ] [ <a href="http://ukpmc.ac.uk/abstract/MED/22976083"><code>ukpmc</code></a> ]
</p>
</blockquote>

<p>
She moved to KCL in 2014, and from there to the CTTV project at Sanger/GSK.
</p>
</div>
</div>

<div id="outline-container-sec-3-7" class="outline-3">
<h3 id="sec-3-7">Nikolas Pontikos (2011-2014)</h3>
<div class="outline-text-3" id="text-3-7">
<img src="pics/NikoSubmitting_small.jpg" alt="portrait" style="float:left;margin:0 5px 0 0;" />
<p>
During Niko's PhD he worked on automated analysis of flow cytometry data and applying
clustering methods to call and impute genotypes in the KIR locus.  His thesis <a href="http://figshare.com/articles/Nikolas_Pontikos_PhD_Thesis_2015/1598198"><i>Normalisation and Clustering Methods Applied to Association Studies in Type 1 Diabetes</i></a> is on figshare.
</p>

<blockquote>
<p>
<b>Key paper</b>: A hybrid qPCR/SNP array approach allows cost efficient assessment of KIR gene copy numbers in large samples. Pontikos et al. <i>BMC genomics</i>, 2014, vol. 15, pp. 274. [ <a href="http://ukpmc.ac.uk/abstract/MED/24720548"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1186/1471-2164-15-274"><code>doi</code></a> ]
</p>
</blockquote>

<p>
He moved to UCL in 2014 and maintains notes about his <a href="http://pontikos.github.io/">current work</a>.  
</p>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="date">Created: 2017-09-12 Tue 09:25</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
