<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>Autoimmunity and statistical genetics</title>
<!-- 2015-02-13 Fri 14:14 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<meta  name="author" content="Chris Wallace" />
<link rel="stylesheet" type="text/css" href="css/mystyle.css" />
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">Autoimmunity and statistical genetics</h1>
<p>
This is a set of pages to store information about datasets and code used in statistical genetics papers from the Wallace group.  For the corporate webpage with publications and details of group members, please go to <a href="http://www-gene.cimr.cam.ac.uk/staff/wallace">http://www-gene.cimr.cam.ac.uk/staff/wallace</a>.  
</p>

<p>
Information that exists here falls under:
</p>

<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#sec-1">Code and data related to papers</a>
<ul>
<li><a href="#sec-1-1">Bowes et al (2015) "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis"</a></li>
<li><a href="#sec-1-2">T1D ImmunoChip paper (in press)</a></li>
<li><a href="#sec-1-3">Liley and Wallace (2015) "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics"</a></li>
<li><a href="#sec-1-4">Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" bioRxiv preprint</a></li>
</ul>
</li>
<li><a href="#sec-2">Datasets we use</a>
<ul>
<li><a href="#sec-2-1">T1D ImmunoChip Data</a></li>
<li><a href="#sec-2-2">ImmunoBase</a></li>
<li><a href="#sec-2-3">Fairfax eQTL data</a></li>
</ul>
</li>
<li><a href="#sec-3">Larger repositories/software packages</a></li>
</ul>
</div>
</div>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1">Code and data related to papers</h2>
<div class="outline-text-2" id="text-1">
</div><div id="outline-container-sec-1-1" class="outline-3">
<h3 id="sec-1-1">Bowes et al (2015) "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis"</h3>
<div class="outline-text-3" id="text-1-1">
<p>
R script for fine mapping <a href="https://github.com/chr1swallace/finemap-psa">https://github.com/chr1swallace/finemap-psa</a> 
RA p values required for input were downloaded from <a href="http://www.immunobase.org">ImmunoBase</a>.
</p>
</div>
</div>

<div id="outline-container-sec-1-2" class="outline-3">
<h3 id="sec-1-2">T1D ImmunoChip paper (in press)</h3>
<div class="outline-text-3" id="text-1-2">
<p>
R code used to perform
 numerical calculations exploring the appropriate p value thresholds
 for calling genetic association in one disease given genomewide
 significant genetic association at the same SNP in a related
 disease. <a href="https://github.com/chr1swallace/condpp">https://github.com/chr1swallace/condpp</a>.  
</p>

<p>
<a href="#sec-2-1">T1D ImmunoChip Data</a>
</p>
</div>
</div>

<div id="outline-container-sec-1-3" class="outline-3">
<h3 id="sec-1-3">Liley and Wallace (2015) "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics"</h3>
<div class="outline-text-3" id="text-1-3">
<p>
Data Availability: GWAS summary statistics (p values) were downloaded from the ImmunoBase website (<a href="http://www.immunobase.org">http://www.immunobase.org</a>). Also uses <a href="#sec-2-1">T1D ImmunoChip Data</a> for simulations to examine properties of the method.
</p>

<p>
James' code is on <a href="https://github.com/jamesliley/cFDR-commo%E2%80%8Bn-controls">https://github.com/jamesliley/cFDR-commo%E2%80%8Bn-controls</a>.
</p>
</div>
</div>

<div id="outline-container-sec-1-4" class="outline-3">
<h3 id="sec-1-4">Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" <a href="http://biorxiv.org/content/early/2015/02/12/015164">bioRxiv preprint</a></h3>
<div class="outline-text-3" id="text-1-4">
<p>
Uses the R package <a href="https://github.com/chr1swallace/GUESSFM">GUESSFM</a>.  For detailed pipeline of steps to run GUESSFM, see the <a href="https://github.com/chr1swallace/GUESSFM/tree/master/vignettes">vignettes</a>.
</p>

<p>
<a href="#sec-2-1">T1D ImmunoChip Data</a>
</p>

<p>
MS data was accessed through application to the <i>International Multiple Sclerosis Genetic Consortium (IMSGC)</i>.
</p>
</div>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2">Datasets we use</h2>
<div class="outline-text-2" id="text-2">
</div><div id="outline-container-sec-2-1" class="outline-3">
<h3 id="sec-2-1">T1D ImmunoChip Data</h3>
<div class="outline-text-3" id="text-2-1">
<p>
from dbGaP (<a href="http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2">http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2</a>)
</p>
</div>
</div>

<div id="outline-container-sec-2-2" class="outline-3">
<h3 id="sec-2-2">ImmunoBase</h3>
<div class="outline-text-3" id="text-2-2">
<p>
Summary ImmunoChip datasets from multiple studies are nicely curated and available from <a href="http://www.immunobase.org">ImmunoBase</a> if you first register (free) then go to <code>Data..downloads</code> on the right, then click through to <code>data/immunochip/association_stats</code>.
</p>
</div>
</div>

<div id="outline-container-sec-2-3" class="outline-3">
<h3 id="sec-2-3">Fairfax eQTL data</h3>
<div class="outline-text-3" id="text-2-3">
<p>
Ben Fairfax has published eQTL papers on sorted cells (monocytes, stimulated monocytes, B cells) from close to 300 samples.  The expression data are freely available, but the genotype data require a data access agreement, available as detailed in either paper.
</p>

<p>
<i>Fairfax et al. (2012) Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502-510</i>
</p>

<p>
gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>.
</p>

<p>
<i>Fairfax et al. (2014) Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science 343, 1246949</i>
</p>

<p>
gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>, <a href="http://www.ebi.ac.uk/ega/EGAD00010000520">http://www.ebi.ac.uk/ega/EGAD00010000520</a>.
</p>
</div>
</div>
</div>

<div id="outline-container-sec-3" class="outline-2">
<h2 id="sec-3">Larger repositories/software packages</h2>
<div class="outline-text-2" id="text-3">
<p>
See <a href="https://github.com/chr1swallace?tab=repositories">https://github.com/chr1swallace?tab=repositories</a> for everything else
</p>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: Chris Wallace</p>
<p class="date">Created: 2015-02-13 Fri 14:14</p>
<p class="creator"><a href="http://www.gnu.org/software/emacs/">Emacs</a> 24.3.1 (<a href="http://orgmode.org">Org</a> mode 8.2.10)</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
