<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>index</title>
<!-- 2015-02-13 Fri 12:50 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<meta  name="author" content="Chris Wallace" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center; }
  .todo   { font-family: monospace; color: red; }
  .done   { color: green; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  pre.src-sh:before    { content: 'sh'; }
  pre.src-bash:before  { content: 'sh'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-R:before     { content: 'R'; }
  pre.src-perl:before  { content: 'Perl'; }
  pre.src-java:before  { content: 'Java'; }
  pre.src-sql:before   { content: 'SQL'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.right  { text-align: center;  }
  th.left   { text-align: center;   }
  th.center { text-align: center; }
  td.right  { text-align: right;  }
  td.left   { text-align: left;   }
  td.center { text-align: center; }
  dt { font-weight: bold; }
  .footpara:nth-child(2) { display: inline; }
  .footpara { display: block; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  /*]]>*/-->
</style>
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">index</h1>
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#sec-1">Code and data related to papers</a>
<ul>
<li><a href="#sec-1-1">Bowes et al (2015) "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis"</a></li>
<li><a href="#sec-1-2">T1D ImmunoChip paper (in press)</a></li>
<li><a href="#sec-1-3">Liley and Wallace (2015) "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics"</a></li>
<li><a href="#sec-1-4">Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" bioRxiv preprint</a></li>
</ul>
</li>
<li><a href="#sec-2">Larger repositories/software packages</a></li>
</ul>
</div>
</div>
<p>
This is a set of pages to store the code I use and related information in my statistical genetic analyses.  For the corporate webpage with publications, please go to <a href="http://www-gene.cimr.cam.ac.uk/staff/wallace">http://www-gene.cimr.cam.ac.uk/staff/wallace</a>.
</p>

<p>
Information that exists here falls under:
</p>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1">Code and data related to papers</h2>
<div class="outline-text-2" id="text-1">
</div><div id="outline-container-sec-1-1" class="outline-3">
<h3 id="sec-1-1">Bowes et al (2015) "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis"</h3>
<div class="outline-text-3" id="text-1-1">
</div><ul class="org-ul"><li><a id="sec-1-1-1" name="sec-1-1-1"></a>Code<br  /><div class="outline-text-4" id="text-1-1-1">
<p>
R script for fine mapping <a href="https://github.com/chr1swallace/finemap-psa">https://github.com/chr1swallace/finemap-psa</a> 
</p>
</div>
</li>

<li><a id="sec-1-1-2" name="sec-1-1-2"></a>Data<br  /><div class="outline-text-4" id="text-1-1-2">
<p>
RA p values downloaded from <a href="http://www.immunobase.org">ImmunoBase</a>.
</p>
</div>
</li></ul>
</div>

<div id="outline-container-sec-1-2" class="outline-3">
<h3 id="sec-1-2">T1D ImmunoChip paper (in press)</h3>
<div class="outline-text-3" id="text-1-2">
</div><ul class="org-ul"><li><a id="sec-1-2-1" name="sec-1-2-1"></a>Code<br  /><div class="outline-text-4" id="text-1-2-1">
<p>
R code used to perform
 numerical calculations exploring the appropriate p value thresholds
 for calling genetic association in one disease given genomewide
 significant genetic association at the same SNP in a related
 disease. <a href="https://github.com/chr1swallace/condpp">https://github.com/chr1swallace/condpp</a>.  
</p>
</div>
</li>

<li><a id="sec-1-2-2" name="sec-1-2-2"></a>Data<br  /><div class="outline-text-4" id="text-1-2-2">
<p>
Will be available from dbGaP at <a href="http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2">http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2</a>
</p>
</div>
</li></ul>
</div>

<div id="outline-container-sec-1-3" class="outline-3">
<h3 id="sec-1-3">Liley and Wallace (2015) "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics"</h3>
<div class="outline-text-3" id="text-1-3">
<p>
Data Availability: GWAS summary statistics (p values) were downloaded from the ImmunoBase website (<a href="http://www.immunobase.org">http://www.immunobase.org</a>). Raw T1D genotype data are available from dbGaP (<a href="http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2">http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2</a>).
</p>
</div>

<ul class="org-ul"><li><a id="sec-1-3-1" name="sec-1-3-1"></a>Code<br  /><div class="outline-text-4" id="text-1-3-1">
<p>
James' code is on <a href="https://github.com/jamesliley/cFDR-commo%E2%80%8Bn-controls">https://github.com/jamesliley/cFDR-commo%E2%80%8Bn-controls</a>.
</p>
</div>
</li></ul>
</div>

<div id="outline-container-sec-1-4" class="outline-3">
<h3 id="sec-1-4">Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" <a href="http://biorxiv.org/content/early/2015/02/12/015164">bioRxiv preprint</a></h3>
<div class="outline-text-3" id="text-1-4">
</div><ul class="org-ul"><li><a id="sec-1-4-1" name="sec-1-4-1"></a>Code<br  /><div class="outline-text-4" id="text-1-4-1">
<p>
Uses the R package <a href="https://github.com/chr1swallace/GUESSFM">GUESSFM</a>.  For detailed pipeline of steps to run GUESSFM, see the <a href="https://github.com/chr1swallace/GUESSFM/tree/master/vignettes">vignettes</a>.
</p>
</div>
</li>

<li><a id="sec-1-4-2" name="sec-1-4-2"></a>Data<br  /><div class="outline-text-4" id="text-1-4-2">
<p>
T1D data will be available from dbGaP at <a href="http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2">http://www.ncbi.nlm.nih.gov/projects/gap%E2%80%8B/cgi-bin/study.cgi?study_id=phs000180.v2%E2%80%8B.p2</a>
</p>

<p>
MS data was accessed through application to the <i>International Multiple Sclerosis Genetic Consortium (IMSGC)</i>
</p>
</div>
</li></ul>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2">Larger repositories/software packages</h2>
<div class="outline-text-2" id="text-2">
<p>
See <a href="https://github.com/chr1swallace?tab=repositories">https://github.com/chr1swallace?tab=repositories</a> for everything else
</p>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: Chris Wallace</p>
<p class="date">Created: 2015-02-13 Fri 12:50</p>
<p class="creator"><a href="http://www.gnu.org/software/emacs/">Emacs</a> 24.3.1 (<a href="http://orgmode.org">Org</a> mode 8.2.10)</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
