#+TITLE: Autoimmunity and statistical genetics
#+options: toc:nil num:nil

This is a set of pages to store information about datasets and code used in statistical genetics papers from Chris Wallace's group.  For the corporate webpage with publications and details of group members, please go to [[http://www-gene.cimr.cam.ac.uk/staff/wallace]].  

Information that exists here falls under:

#+TOC: headlines 2

* Code and data related to papers


** Bowes et al (2015) "Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis"

R script for fine mapping https://github.com/chr1swallace/finemap-psa 
RA p values required for input were downloaded from [[http://www.immunobase.org][ImmunoBase]].

** T1D ImmunoChip paper (in press)

 R code used to perform
  numerical calculations exploring the appropriate p value thresholds
  for calling genetic association in one disease given genomewide
  significant genetic association at the same SNP in a related
  disease. https://github.com/chr1swallace/condpp.  

[[T1D ImmunoChip Data]]

** Liley and Wallace (2015) "A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics"

Data Availability: GWAS summary statistics (p values) were downloaded from the ImmunoBase website ([[http://www.immunobase.org]]). Also uses [[T1D ImmunoChip Data]] for simulations to examine properties of the method.

James' code is on https://github.com/jamesliley/cFDR-commo​n-controls.

** Wallace et al "Dissection of a complex disease susceptibility region using a Bayesian stochastic search approach to fine mapping" [[http://biorxiv.org/content/early/2015/02/12/015164][bioRxiv preprint]]

Uses the R package [[https://github.com/chr1swallace/GUESSFM][GUESSFM]].  For detailed pipeline of steps to run GUESSFM, see the [[https://github.com/chr1swallace/GUESSFM/tree/master/vignettes][vignettes]].

[[T1D ImmunoChip Data]]

MS data was accessed through application to the [[International Multiple Sclerosis Genetic Consortium][International Multiple Sclerosis Genetic Consortium (IMSGC)]].

* Datasets we use

** T1D ImmunoChip Data 

from dbGaP ([[http://www.ncbi.nlm.nih.gov/projects/gap​/cgi-bin/study.cgi?study_id=phs000180.v2​.p2]])

** ImmunoBase

Summary ImmunoChip datasets from multiple studies are nicely curated and available from [[http://www.immunobase.org][ImmunoBase]] if you first register (free) then go to =Data..downloads= on the right, then click through to =data/immunochip/association_stats=.

** Fairfax eQTL data 

Ben Fairfax has published eQTL papers on sorted cells (monocytes, stimulated monocytes, B cells) from close to 300 samples.  The expression data are freely available, but the genotype data require a data access agreement, available as detailed in either paper.

/Fairfax et al. (2012) Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502-510/

gene expression data: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945; genotyping data: http://www.ebi.ac.uk/ega/EGAD00010000144.

/Fairfax et al. (2014) Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science 343, 1246949/

gene expression data: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232; genotyping data: http://www.ebi.ac.uk/ega/EGAD00010000144, http://www.ebi.ac.uk/ega/EGAD00010000520.

* Larger repositories/software packages

See [[https://github.com/chr1swallace?tab=repositories]] for everything else
