<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<title>Research</title>
<!-- 2017-01-25 Wed 19:42 -->
<meta  http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta  name="generator" content="Org-mode" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center; }
  .todo   { font-family: monospace; color: red; }
  .done   { color: green; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: visible;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline;}
  pre.src-sh:before    { content: 'sh'; }
  pre.src-bash:before  { content: 'sh'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-R:before     { content: 'R'; }
  pre.src-perl:before  { content: 'Perl'; }
  pre.src-java:before  { content: 'Java'; }
  pre.src-sql:before   { content: 'SQL'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.right  { text-align: center;  }
  th.left   { text-align: center;   }
  th.center { text-align: center; }
  td.right  { text-align: right;  }
  td.left   { text-align: left;   }
  td.center { text-align: center; }
  dt { font-weight: bold; }
  .footpara:nth-child(2) { display: inline; }
  .footpara { display: block; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  /*]]>*/-->
</style>
<link rel="stylesheet" type="text/css" href="css/poole.css"/>
<link rel="stylesheet" type="text/css" href="css/lanyon.css"/>
<link rel="stylesheet" type="text/css" href="css/tabs.css"/>
<script type="text/javascript" src="styles/bigblow/js/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery-ui-1.10.2.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.localscroll-min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.scrollTo-1.4.3.1-min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/jquery.zclip.min.js"></script>
<script type="text/javascript" src="styles/bigblow/js/bigblow.js"></script>
<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
<script type="text/javascript">
/*
@licstart  The following is the entire license notice for the
JavaScript code in this tag.

Copyright (C) 2012-2013 Free Software Foundation, Inc.

The JavaScript code in this tag is free software: you can
redistribute it and/or modify it under the terms of the GNU
General Public License (GNU GPL) as published by the Free Software
Foundation, either version 3 of the License, or (at your option)
any later version.  The code is distributed WITHOUT ANY WARRANTY;
without even the implied warranty of MERCHANTABILITY or FITNESS
FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.

As additional permission under GNU GPL version 3 section 7, you
may distribute non-source (e.g., minimized or compacted) forms of
that code without the copy of the GNU GPL normally required by
section 4, provided you include this license notice and a URL
through which recipients can access the Corresponding Source.


@licend  The above is the entire license notice
for the JavaScript code in this tag.
*/
<!--/*--><![CDATA[/*><!--*/
 function CodeHighlightOn(elem, id)
 {
   var target = document.getElementById(id);
   if(null != target) {
     elem.cacheClassElem = elem.className;
     elem.cacheClassTarget = target.className;
     target.className = "code-highlighted";
     elem.className   = "code-highlighted";
   }
 }
 function CodeHighlightOff(elem, id)
 {
   var target = document.getElementById(id);
   if(elem.cacheClassElem)
     elem.className = elem.cacheClassElem;
   if(elem.cacheClassTarget)
     target.className = elem.cacheClassTarget;
 }
/*]]>*///-->
</script>
</head>
<body>
<div id="content">
<h1 class="title">Research</h1>
    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior.  -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">
    <label for="sidebar-checkbox" class="sidebar-toggle"></label>    


<!-- Toggleable sidebar -->
    <div class="sidebar" id="sidebar">
      <div class="sidebar-item">
        <p>Chris Wallace group:<br/><b>statistics</b>, applied to<br/> 
        <b>genomics</b>, to understand<br/>
        <b>autoimmune diseases</b>.</p>
      </div>
      
      <nav class="sidebar-nav">
        <a class="sidebar-nav-item" href="index.html"><i class="fa fa-home"></i></a>
        <!-- <a href="./background.html">Background</a> |  -->
        <a class="sidebar-nav-item" href="./group.html">People</a>
        <a class="sidebar-nav-item" href="./research.html">Research</a>
        <a class="sidebar-nav-item" href="./data.html">Data</a>
        <a class="sidebar-nav-item" href="./code.html">Software</a>
        <!-- <a class="sidebar-nav-item" href="./vacancies.html">Vacancies</a> -->
        <!-- <a class="sidebar-nav-item" href="http://europepmc.org/search?query=AUTHORID:%220000-0001-9755-1703%22&sortby=Date">Publications</a> | -->
        <a class="sidebar-nav-item" href="./pubs2.html">Publications</a>
        <a class="sidebar-nav-item" href="./contact.html">Links and contact</a>
      </nav>
    </div>

<div id="outline-container-sec-1" class="outline-2">
<h2 id="sec-1">Causal variants</h2>
<div class="outline-text-2" id="text-1">
<img src="GUESSFM_small.png" alt="portrait" style="float:left;margin:10px;" />

<p>
Genetic association studies have identified many DNA sequence variants
that associate with disease risk. By exploiting the known correlation
that exists between neighbouring variants in the genome, inference can
be extended beyond those individual variants tested to identify sets
within which a causal variant is likely to reside. However, this
correlation, particularly in the presence of multiple disease causing
variants in relative proximity, makes disentangling the specific
causal variants difficult. Statistical approaches to this fine mapping
problem have traditionally taken a stepwise search approach, beginning
with the most associated variant in a region, then iteratively
attempting to find additional associated variants. We adapted a
stochastic search approach that avoids this stepwise process and is
explicitly designed for dealing with highly correlated predictors to
the fine mapping problem. We showed in simulated data that it
outperforms its stepwise counterpart and other variable selection
strategies such as the lasso. We applied our approach to understand
the association of two immune-mediated diseases to a region on
chromosome 10p15. We identified a model for multiple sclerosis
containing two variants, neither of which was found through a stepwise
search, and functionally linked both of these to the neighbouring
candidate gene, IL2RA, in independent data. Our approach can be used
to aid fine mapping of other disease-associated regions, which is
critical for design of functional follow-up studies required to
understand the mechanisms through which genetic variants influence
disease.
</p>
</div>

<div id="outline-container-sec-1-1" class="outline-3">
<h3 id="sec-1-1">Related papers</h3>
<div class="outline-text-3" id="text-1-1">
<p>
Dissection of a Complex Disease Susceptibility Region Using a Bayesian Stochastic Search Approach to Fine Mapping. Wallace et al., <i>PLoS genetics</i> 2015.  [ <a href="http://ukpmc.ac.uk/abstract/MED/26106896"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1371/journal.pgen.1005272"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>
</div>
</div>
</div>

<div id="outline-container-sec-2" class="outline-2">
<h2 id="sec-2">Causal genes</h2>
<div class="outline-text-2" id="text-2">
<p>
Genomewide association studies (GWAS) have been hugely successful in
identifying associations between genetic variation and risk of common
diseases.  However, genetic variation will not be a pharmaceutical
target for these disease and translating this knowledge into
understanding the genes, cells and pathways involved in disease
aetiology has been slow.  This is because the associated genetic
variants do not typically reside in genes and change the protein they
encode, but lie between genes and are presumed to regulate their
expression in some cells, perhaps under specific conditions.
</p>
<img src="ctsh.jpg" alt="portrait" style="float:right;margin:10px;" />

<p>
As GWAS studies for disease have been conducted over the past decade,
so people have conducted GWAS for other, more gene specific traits,
such as parallel GWAS for the expression levels of each of the ~20,000
protein coding genes in a given cell type, called "eQTL" studies.
These produce a local GWAS trace just as a disease GWAS does.  We use
a pattern matching technique called colocalisation to determine
whether the two GWAS traces are compatible with the two traits,
disease risk and expression of a given gene in a given cell type and
condition, sharing a causal variant.
</p>


<img src="pchic.png" alt="portrait" style="float:left;margin:10px;" />

<p>
Another approach is to exploit knowledge about the 3D folding of
chromatin derived from high throughput Chromosome Conformation Capture
(Hi-C) in its targetted form: Capture Hi-C (CHi-C).  This allows us to link GWAS causal variants (mapped probabilistically) to the genes they regulate, and we have deployed this approach across 17 primary human sorted cell types.
</p>
</div>


<div id="outline-container-sec-2-1" class="outline-3">
<h3 id="sec-2-1">Related papers</h3>
<div class="outline-text-3" id="text-2-1">
<p>
Statistical colocalization of monocyte gene expression and genetic risk variants for type 1 diabetes. Wallace et al., <i>Human Molecular Genetics</i> 2012, vol. 21, no. 12, pp. 2815-2824.  [ <a href="http://dx.doi.org/10.1093/hmg/dds098"><code>doi</code></a> ] [ <a href="http://ukpmc.ac.uk/abstract/MED/22403184"><code>ukpmc</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>

<p>
Statistical testing of shared genetic control for potentially related traits. Wallace. <i>Genetic Epidemiology</i> 2013, vol. 37, no. 8, pp. 802-813.  [ <a href="http://dx.doi.org/10.1002/gepi.21765"><code>doi</code></a> ] [  <i class="fa fa-unlock-alt" aria-hidden="true"></i>  <a href="http://ukpmc.ac.uk/abstract/MED/24227294"><code>ukpmc</code></a> ]
</p>

<p>
A Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. Giambartolomei et al., <i>PLoS genetics</i> 2014.  [ <a href="http://ukpmc.ac.uk/abstract/MED/24830394"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1371/journal.pgen.1004383"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>

<p>
Integration of disease association and eQTL data using a Bayesian colocalisation approach highlights six candidate causal genes in immune-mediated diseases. Guo et al., <i>Human Molecular Genetics</i> 2015.  [ <a href="http://ukpmc.ac.uk/abstract/MED/25743184"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1093/hmg/ddv077"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>

<p>
Chromosome contacts in activated T cells identify autoimmune disease-candidate genes. Burren et al., bioRxiv 100958 [ <a href="https://doi.org/10.1101/100958"><code>doi</code></a> ] <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>

<p>
Lineage-Specific Genome Architecture Links Enhancers and Non-coding Disease Variants to Target Gene Promoters. Javierre et al., Cell 2016 [ <a href="http://ukpmc.ac.uk/abstract/MED/27863249"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1016/j.cell.2016.09.037"><code>doi</code></a> ] <i class="fa fa-unlock-alt" aria-hidden="true"></i>
</p>
</div>
</div>
</div>


<div id="outline-container-sec-3" class="outline-2">
<h2 id="sec-3">Diseases, their relationships and divisions</h2>
<div class="outline-text-2" id="text-3">
<img src="disease-network.png" alt="portrait" style="float:left;margin:10px;" />

<p>
We study the shared and distinct genetic aetiology between related
autoimmune diseases both at the genomewide and region specific level,
addressing two separate questions.  First, at a the level of the whole
genome, do associations with one disease predict associations with
another disease?  We do this using an approach which estimates
conditional false discovery rates for association of each disease
at each SNP conditional on the other disease.  Because this uses
genomewide associations, it includes information from individual loci
that are not genomewide significant, but show a trend to shared
association for two diseases.  However, because genes with similar
function are often located close to one another, and because genetic
variants also show a spatial correlation due to linkage
disequilibrium, this does not autoimattically imply that the two
diseases share the same causal variants.  We therefore use a
complementary approach, applying the an adapted colocalisation method that we use for matching disease and gene expression signatures to investigate sharing of causal variants in detail.
This shows that even for two apparently closely related autoimmune
diseases which both show association to the same genetic region
(delineated by two recombination hotspots), this will relate to them
sharing the same causal variant only about 50% of the time.
</p>

<p>
More recently, we have developed a genome-wide test for determining whether some phenotypically or clinically defined division of patients corresponds to a genetic signature of aetiological heterogeneity.
</p>
</div>

<div id="outline-container-sec-3-1" class="outline-3">
<h3 id="sec-3-1">Related papers</h3>
<div class="outline-text-3" id="text-3-1">
<p>
A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics. Liley and Wallace, <i>PLoS Genetics</i> 2015. [ <a href="http://ukpmc.ac.uk/abstract/MED/25658688"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1371/journal.pgen.1004926"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>

<p>
Fine mapping of type 1 diabetes susceptibility loci and evidence for colocalization of causal variants with lymphoid gene enhancers. Onengut-Gumuscu et al., <i>Nature Genetics</i> 2015 [ <a href="http://ukpmc.ac.uk/abstract/MED/25751624"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ng.3245"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>

<p>
Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls. Fortune et al., <i>Nature Genetics</i> 2015.  [ <a href="http://ukpmc.ac.uk/abstract/MED/26053495"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ng.3330"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>
<p>
A method for identifying genetic heterogeneity within phenotypically defined disease subgroups. Liley et al., <i>Nature Genetics</i> 2016.  [ <a href="http://europepmc.org/abstract/MED/28024155"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ng.3751"><code>doi</code></a> ] [ <a href="https://doi.org/10.1101/100958"><code>biorxiv</code></a> <i class="fa fa-unlock-alt" aria-hidden="true"></i> ]
</p>
</div>
</div>
</div>

<div id="outline-container-sec-4" class="outline-2">
<h2 id="sec-4">Gene expression in immune cells</h2>
<div class="outline-text-2" id="text-4">
<img src="ferreira.jpg" style="float:right;margin:10px;" />

<p>
The BABYDIET study led by Annette Ziegler and Ezio Bonifacio collected
blood samples longitudinally from 109 children genetically at risk of,
but initially unaffected by type 1 diabetes.  We were lucky to have
access to white blood cells (PBMCs) from these samples and measured
the gene expression in them.  We were particularly interested in
expression of interferon responsive genes, given previous links
between type 1 diabetes and infection.  Indeed, we saw an upregulation
of these genes in children who went on to develop the autoantibodies
that are strongly predictive of T1D diagnosis, but, crucially, the
upregulation was transient and preceded the appearance of
autoantibodies.  It was also temporally correlated with recent upper
respiratory infection, and may represent a biomarker for the response
to infection or the mechanism by which the infection influences type 1
diabetes risk.
</p>

<p>
We also used these data to investigate seasonal variation in gene
expression, and found that 25% of the genes expressed in these cells
in these children varied in their expression throughout the year.  In
winter, we saw that this expression profile produced a
pro-inflammatory environment.  This might be advantageous during a
season when infectious diseases are at a peak, but is a risk factor
for other diseases associated with inflammation such as cardiovascular
also peak in winter.  We replicated this finding in multiple datasets,
including one from the Southern hemisphere when winter occurs during
June-August.
</p>
</div>

<div id="outline-container-sec-4-1" class="outline-3">
<h3 id="sec-4-1">Related papers</h3>
<div class="outline-text-3" id="text-4-1">
<p>
A type I interferon transcriptional signature precedes autoimmunity in children genetically at-risk of type 1 diabetes. Ferreira et al. <i>Diabetes</i> 2014.  [ <a href="http://ukpmc.ac.uk/abstract/MED/24561305"><code>ukpmc</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>

<p>
Widespread seasonal gene expression reveals annual differences in human immunity and physiology. Dopico et al. <i>Nature Communications</i> 2015  [ <a href="http://ukpmc.ac.uk/abstract/MED/25965853"><code>ukpmc</code></a> ] [ <a href="http://dx.doi.org/10.1038/ncomms8000"><code>doi</code></a> ]  <i class="fa fa-unlock-alt" aria-hidden="true"></i> 
</p>
</div>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="date">Created: 2017-01-25 Wed 19:42</p>
<p class="validation"><a href="http://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
