<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Group</title>
  <meta name="description" content="Statistics applied to genomics to understand autoimmune diseases">

  <link rel="stylesheet" href="/assets/main.css">
  <link rel="stylesheet" href="/assets/css/style.css">
  <link rel="canonical" href="http://localhost:4000/group/">
  <link rel="alternate" type="application/rss+xml" title="Wallace group" href="/feed.xml">
  
  
</head>


  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Wallace group</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
           
	  <a class="page-link" style="color:#099;font-weight:bold;" href="/group/">Group</a>
	  
            
          
            
            
          
            
            
           
	  <a class="page-link" href="/data/">Data</a>
	  
            
          
            
            
          
            
            
          
            
            
           
	  <a class="page-link" href="/papers/">Papers</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/research/">Research</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/code/">Code</a>
	  
            
          
            
            
          
            
            
           
	  <a class="page-link" href="/talks/">Presentations</a>
	  
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <!-- <header class="post-header"> -->
  <!--   <h1 class="post-title">Group</h1> -->
  <!-- </header> -->

  <div class="post-content">
    <h1 id="chris-wallace">Chris Wallace</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/pic_200.jpg" alt="portait" /></p>
</div>
<p>I am a statistician who works with biological datasets to understand
the mechanisms underlying human disease and identify possible
treatments.  Such work is only possible with close collaboration with
biologists, and I moved to the <a href="http://www.cimr.cam.ac.uk">Cambridge
Institute for Medical Research</a> in 2009 to pursue these
collaborations, most closely with the <a href="http://www-gene.cimr.cam.ac.uk">Diabetes and Inflammation
Laboratory</a>.  During
the GWAS era, we, and others, identified many genetic polymorphisms
that alter risk of human diseases such as type 1 diabetes and
hypertension.  My focus has now mostly shifted towards understanding
the cell specific mechanism through which these variants affect risk
of different immune mediated diseases and intermediate biological
traits such as gene expression.</p>

<p>Statistically, my current work is focused on variable
selection, Bayesian model averaging and empirical Bayes false
discovery rates, applied to integrated analysis of multiple related
datasets.</p>

<p>My aim for my group is that, by integrating detailed biological understanding with excellent statistical inference we will make a meaningful contribution to the understanding of autoimmune diseases.</p>

<p>In Feburary 2016 I moved to the <a href="http://www.med.cam.ac.uk/">Department of Medicine</a> as a PI with the aim of establishing new collaborations. I am funded by the <a href="http://www.wellcome.ac.uk/">Wellcome Trust</a> as a Senior Research Fellow  and hold an honorary Programme Leader  position  at the <a href="http://www.mrc-bsu.cam.ac.uk/">MRC Biostatistics Unit</a>.</p>

<p><em>Previously</em></p>
<ul>
  <li>2009-2016 :: Wellcome Trust Career Development Fellow in the  <a href="http://www-gene.cimr.cam.ac.uk">Diabetes and Inflammation Laboratory</a> (and head of the DIL stats group from 2012)</li>
  <li>2003-2008 :: Research Fellow working with	  the <a href="http://www.brightstudy.ac.uk/">BRIGHT</a> study into	  the genetics of hypertension.</li>
  <li>1999-2003 :: PhD: <a href="http://figshare.com/articles/Human_genetic_susceptibility_to_leprosy_methodological_issues_in_a_linkage_analysis_of_extended_pedigrees_from_Karonga_district_Malawi/963086">Genetic Susceptibility to leprosy: methodological issues in a linkage  analysis of extended pedigrees from Karonga District, Malawi</a>,	  <a href="http://www.lshtm.ac.uk">LSHTM</a>, part of the	  <a href="http://www.lshtm.ac.uk/eph/ide/research/kps">Karonga Prevention Study</a>.</li>
  <li>1997-1999 :: Statistical programmer, <a href="http://www.imperial.ac.uk">Imperial</a>.</li>
</ul>

<hr />

<h2 id="current-members">Current members</h2>

<h1 id="jennifer-asimit">Jennifer Asimit</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/Jennifer_small.jpg" alt="portait" /></p>
</div>
<p>Jenn Asimit is a senior investigator statistician with an interest in statistical methodology development for the analysis of multiple traits, as well as fine-mapping and rare variant association approaches. She recently held a MRC Methodology Research fellowship at the Sanger institute and has developed several methods and software for rare variant association analyses and the overlap analysis of traits.</p>

<div style="clear:both"></div>

<h1 id="oliver-burren">Oliver Burren</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/Olly_small.jpg" alt="portait" /></p>
</div>
<p><a href="http://ollyburren.github.io">Olly Burren</a> is undertaking a PhD, researching methods to integrate genomic and genetic datasets to better understand autoimmune disease mechanisms. He was previously head of Genome Informatics in the <a href="https://www-gene.cimr.cam.ac.uk/">Diabetes and Inflammation Laboratory</a> where he was responsible for the creation and smooth running of the <a href="http://www.immunobase.org">ImmunoBase</a>,  <a href="http://www.t1dbase.org">T1DBase</a> and <a href="http://www.chicp.org">CHiCP</a> web resources which collate data and summary information from genomic, GWAS and ImmunoChip analyses of autoimmune diseases.  He continues to maintain  <a href="http://www.chicp.org">CHiCP</a> as a means of visualizing Capture Hi-C datasets.</p>

<div style="clear:both"></div>

<h1 id="alessandra-cabassi">Alessandra Cabassi</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/alessandra_150.jpg" alt="portait" /></p>
</div>
<p>Alessandra Cabassi is a PhD student at the MRC Biostatistics Unit, jointly supervised by <a href="http://www.mrc-bsu.cam.ac.uk/people/in-alphabetical-order/h-to-m/paul-kirk/">Paul Kirk</a>. She is working on Bayesian nonparametric approaches for integrative modelling of genomic “big data”.</p>

<div style="clear:both"></div>

<h1 id="nastasiya-grinberg">Nastasiya Grinberg</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/nfg_small.jpg" alt="portait" /></p>
</div>
<p>Nastasiya Grinberg is a Wellcome Trust funded postdoctoral researcher. Her research interests revolve around applications of machine learning and data mining techniques to GWAS data and in bioinformatics. She also has a special interest in scientific computing.</p>

<div style="clear:both"></div>

<h1 id="james-liley">James Liley</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/jamesliley_small.jpg" alt="portait" /></p>
</div>
<p>James Liley is a NIHR/BRC funded student on the Wellcome Trust four-year PhD programme in Mathematical Genomics and Medicine. He is working on analysis of GWAS and similar data across multiple diseases and in disease subtypes.</p>

<div style="clear:both"></div>

<hr />

<h2 id="alumni">Alumni</h2>

<h1 id="loes-rutten-jacobs">Loes Rutten-Jacobs</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/Loes_small.jpg" alt="portait" /></p>
</div>
<p><a href="http://www.neuroscience.cam.ac.uk/directory/profile.php?lr406">Loes Rutten-Jacobs</a> is a British Heart Foundation Immediate Research Fellow in the <a href="http://www.neurology.cam.ac.uk/neurology-unit-research-groups/stroke-research-group/">Stroke Research Group</a> of the Department of Clinical Neurosciences, University of Cambridge and is visiting our group until December 2018. In her research she aims to identify novel pathophysiological mechanisms underlying cerebral small vessel disease, using genetic and epidemiologic approaches.</p>

<div style="clear:both"></div>

<h1 id="chris-eijsbouts-2016-2017">Chris Eijsbouts (2016-2017)</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/ChrisE.jpg" alt="portait" /></p>
</div>
<p>Chris Eijsbouts first joined the group as an MPhil student in Computational Biology for a 3 month summer project.  He stayed on afterwards as a research assistant, to continue his work on the interpretation of Capture Hi-C data. The goal of this project was to develop a method for improved identification of chromatin contacts which regulate gene expression, thereby facilitating the interpretation of GWAS studies.
He will start a PhD in Oxford in Autumn 2017.</p>

<div style="clear:both"></div>

<h1 id="steven-kiddle-2014-2017">Steven Kiddle (2014-2017)</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/kiddle_small.jpg" alt="portait" /></p>
</div>
<p><a href="https://kclpure.kcl.ac.uk/portal/steven.kiddle.html">Steven Kiddle</a> held an MRC Career Development Fellow in Biostatistics based at King’s College London, on which Chris Wallace was a statistical mentor. He visited our group in Cambridge from 2015-6, working on clustering approaches for time series and biomarkers of Alzheimer’s Disease.</p>

<p>He now holds a new MRC Career Development Fellowship and is based in the  <a href="http://www.mrc-bsu.cam.ac.uk">MRC Biostatistics Unit</a>.</p>

<div style="clear:both"></div>

<h1 id="mary-fortune-2013-2017">Mary Fortune (2013-2017)</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/mary_150.jpg" alt="portait" /></p>
</div>
<p><a href="http://www.damtp.cam.ac.uk/people/mdf34/">Mary Fortune</a>  worked as a Wellcome Trust PhD student in the <a href="http://cato.medschl.cam.ac.uk/graduate-programmes/phd-programmes/wellcome-trust-mathematical-genomics-and-medicine-mgm/">Mathematical Genomics and Medicine</a> programme on colocalisation analysis in dependent datasets and methods for fitting models of genetic causality to summary GWAS data. She stayed on in the group to complete papers arising from her PhD work and has now moved to the <a href="http://www.mrc-bsu.cam.ac.uk">MRC Biostatistics Unit</a>.</p>

<p>Thesis: <a href="https://doi.org/10.17863/CAM.10463"><em>Statistical Techniques to Fine Map the Related Genetic Aetiology of Autoimmune Diseases</em></a></p>

<!-- Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls. Fortune et al. /Nature Genetics/ 2015, vol. 47, no. 7, pp. 839-846.  [ [=ukpmc=](http://ukpmc.ac.uk/abstract/MED/26053495) ] [ [=doi=](http://dx.doi.org/10.1038/ng.3330) ] -->

<div style="clear:both"></div>

<h1 id="lucia-cilloni-summer-2015">Lucia Cilloni (Summer 2015)</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/LuciaCilloni.png" alt="portait" /></p>
</div>
<p>Lucia did her BSc in Mathematics at the University of Newcastle Upon
Tyne and her MPhil in Computational Biology at
Cambridge University. She did her summer project in the Wallace group on
eQTL-GWAS colocalisation analysis to identify causal genes for Type 1
Diabetes. She is now studying for a PhD in Epidemiology at
Imperial College London.</p>

<div style="clear:both"></div>

<h1 id="hui-guo-2010-2014">Hui Guo (2010-2014)</h1>

<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/hui.jpg" alt="portait" /></p>
</div>
<p>Hui worked as a post doctoral researcher on gene expression data, and led the analysis of our study relating gene expression in PBMCs from children at risk of type 1 diabetes to autoantibody positivity.  She is now a lecturer in biostatistics at <a href="http://www.population-health.manchester.ac.uk/staff/HuiGuo/">University of Manchester</a>.</p>

<div style="float:left;margin:0 5px 0 0;">

  <h1 id="xin-yang-2011-2014">Xin Yang (2011-2014)</h1>
  <div style="float:left;margin:0 5px 0 0;">
    <p><img src="/assets/xin.jpg" alt="portait" /></p>
  </div>
  <p>Xin’s PhD focused on analysis of all kinds of next gen sequencing data - pooled DNA, RNA
seq, and bisulfite seq.  She had a special interest in modelling the extra binomial
variation induced by pooling.</p>

  <p>Thesis: <a href="http://figshare.com/articles/Statistical_analysis_of_next_generation_sequencing_data_with_applications_to_type_1_diabetes_studies/1598221"><em>Statistical analysis of next-generation sequencing data with applications to type 1 diabetes studies</em></a> is on figshare.</p>

  <!-- *Key paper*: Extra-binomial variation approach for analysis of pooled DNA sequencing data. Yang et al. /Bioinformatics/ 2012, vol. 28, no. 22, pp. 2898-2904. [ [=doi=](http://dx.doi.org/10.1093/bioinformatics/bts553) ] [ [=ukpmc=](http://ukpmc.ac.uk/abstract/MED/22976083) ] -->

  <p>She moved to KCL in 2014, and from there to the CTTV project at Sanger/GSK.</p>

  <div style="clear:both"></div>

  <h1 id="nikolas-pontikos-2011-2014">Nikolas Pontikos (2011-2014)</h1>

  <div style="float:left;margin:0 5px 0 0;">
    <p><img src="/assets/NikoSubmitting_small.jpg" alt="portait" /></p>
  </div>
  <p>During Niko’s PhD he worked on automated analysis of flow cytometry data and applying
clustering methods to call and impute genotypes in the KIR locus.</p>

  <p>Thesis: <a href="http://figshare.com/articles/Nikolas_Pontikos_PhD_Thesis_2015/1598198"><em>Normalisation and Clustering Methods Applied to Association Studies in Type 1 Diabetes</em></a></p>

  <!-- *Key paper*: A hybrid qPCR/SNP array approach allows cost efficient assessment of KIR gene copy numbers in large samples. Pontikos et al. /BMC genomics/, 2014, vol. 15, pp. 274. [ [=ukpmc=](http://ukpmc.ac.uk/abstract/MED/24720548) ] [ [=doi=](http://dx.doi.org/10.1186/1471-2164-15-274) ] -->

  <p>He moved to UCL in 2014 and maintains notes about his <a href="http://pontikos.github.io/">current work</a>.</p>

</div>

  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!-- <h2 class="footer-heading">Wallace group</h2> -->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Chris Wallace
            
            </li>
	  <li><a href="/contact/">Contact details</a>
            <!--  -->
            <!-- <li><a href="mailto:cew54-at-cam-dot-ac-dot-uk">cew54-at-cam-dot-ac-dot-uk</a></li> -->
            <!--  -->
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/chr1swallace"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/chr1swallace"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

        </ul>
      </div>

      <div class="footer-col footer-col-1">
        <p>Statistics applied to genomics to understand autoimmune diseases</p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
